lib/api.ts

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: process.env.NEXT_PUBLIC_BASE_URL,
+});
+
+export type SimpleItem = {
+  isCompleted: boolean;
+  name: string;
+  id: number;
+};
+
+export type ResponseSimpleItems = SimpleItem[];
+
+export type ResponseItem = {
+  isCompleted: boolean;
+  imageUrl: string | null;
+  memo: string | null;
+  name: string;
+  tenantId: string;
+  id: number;
+};
```

Comment:
💊 제안
SimpleItem 타입을 별도로 재선언하기보다는 Pick을 활용해 기존 타입에서 필요한 속성만 추출해 재사용하시면 가독성과 유지보수 측면에서 더 좋을 것 같아요.

```suggestion
export type ResponseItem = {
  isCompleted: boolean;
  imageUrl: string | null;
  memo: string | null;
  name: string;
  tenantId: string;
  id: number;
};

export type SimpleItem = Pick<ResponseItem, "id" | "name" | "isCompleted">;

export type ResponseSimpleItems = SimpleItem[];
```